apps/condo/domains/user/integration/telegram/routes.js

```diff
@@ -0,0 +1,163 @@
+const { generators } = require('openid-client')
+
+const conf = require('@open-condo/config')
+const { fetch } = require('@open-condo/keystone/fetch')
+const { getLogger } = require('@open-condo/keystone/logging')
+const { getRedisClient } = require('@open-condo/keystone/redis')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+
+const {
+    TELEGRAM_AUTH_STATUS_PENDING,
+    TELEGRAM_AUTH_STATUS_ERROR,
+    TELEGRAM_AUTH_STATUS_SUCCESS,
+    TELEGRAM_AUTH_REDIS_TTL,
+} = require('@condo/domains/user/integration/telegram/constants')
+const {
+    getUserType,
+    getRedisSessionKey,
+    decodeIdToken,
+    parseJson,
+    validateTelegramAuthConfig,
+    getAuthLink,
+    signUniqueKey,
+    verifyUniqueKey,
+} = require('@condo/domains/user/integration/telegram/utils')
+
+const { syncUser } = require('./sync/syncUser')
+const { startAuthedSession } = require('./utils')
+
+const TELEGRAM_AUTH_CONFIG = conf.TELEGRAM_AUTH_CONFIG ? JSON.parse(conf.TELEGRAM_AUTH_CONFIG) : {}
+const logger = getLogger('telegram-auth')
+const redisClient = getRedisClient()
```

Comment:
You need to rebase, this util does not exist not...
```suggestion
const kv = getKVClient()
```